app/src/main/java/xyz/jhughes/laundry/LaundryParser/MachineTypes.java

```diff
@@ -0,0 +1,11 @@
+package xyz.jhughes.laundry.LaundryParser;
+
+/**
+ * Created by dmtsc on 10/18/2016.
+ */
+
+public class MachineTypes {
+    public static final String DRYER = "Dryer";
```

Comment:
I really do want to have this in Strings.xml. The issue is that there isnt always a context available when we want access to these. 

I also want an overhaul of things a bit .